### PR TITLE
added another degrib2 test

### PR DESCRIPTION
**Pull request by edwardhartnett**
Added more testing.

In this PR I expand the degrib2 testing in run_all_ftp_tests.sh to include more of the test files available on the FTP site.

Part of #116

### CODE REVIEW

File: tests/CMakeLists.txt [hunk 1]
```diff
@@ -92,14 +102,18 @@ endif()
...
+
+  set(FTP_TEST_FILES WW3_Regional_US_West_Coast_20220718_0000.grib2
+    WW3_Regional_US_East_Coast_20220717_0600.grib2
+    gdas.t12z.pgrb2.1p00.anl.grib2 blend.t19z.core.f001.co.grib2
+    cmc_geavg.t12z.pgrb2a.0p50.f000 flxf2022111712.01.2022111712.grb2
+    GLOBAL.grib2.2022103000.0000 hiresw.t00z.arw_5km.f00.hi.grib2
+    naefs_ge10pt.t12z.pgrb2a.0p50_bcf003 rap.t00z.awp130pgrbf00.grib2
+    seaice.t00z.grb.grib2 sgx_nwps_CG3_20221117_1200.grib2
+    GLOBAL.grib2.2022103000.0000 hiresw.t00z.arw_5km.f00.hi.grib2
+    naefs_ge10pt.t12z.pgrb2a.0p50_bcf003 rap.t00z.awp130pgrbf00.grib2
```
Comment (AlexanderRichert-NOAA) [inline tests/CMakeLists.txt]: Looks like there are some duplicates here: GLOBAL.grib2.2022103000.0000, hiresw.t00z.arw_5km.f00.hi.grib2, naefs_ge10pt.t12z.pgrb2a.0p50_bcf003, rap.t00z.awp130pgrbf00.grib2, sgx_nwps_CG3_20221117_1200.grib2

File: tests/run_all_ftp_tests.sh [hunk 2]
```diff
@@ -9,13 +9,24 @@ set -e
...
+hiresw.t00z.arw_5km.f00.hi.grib2 \
+naefs_ge10pt.t12z.pgrb2a.0p50_bcf003 \
+rap.t00z.awp130pgrbf00.grib2 \
+seaice.t00z.grb.grib2 \
+sgx_nwps_CG3_20221117_1200.grib2 \
+cmc_geavg.t12z.pgrb2a.0p50.f000 \
+WW3_Regional_US_West_Coast_20220718_0000.grib2 \
+WW3_Regional_US_East_Coast_20220717_0600.grib2 \
+gdas.t12z.pgrb2.1p00.anl.grib2 \
+flxf2022111712.01.2022111712.grb2 \
```
Comment (AlexanderRichert-NOAA) [inline tests/run_all_ftp_tests.sh:23]: Is there some way to use the list from CMake to get ride of the redundancy here? If not no worries

File: tests/CMakeLists.txt (hunk 1 above)
Comment (edwardhartnett) [reply tests/CMakeLists.txt]: Good catch! I fixed it. THere should be no duplicates.

File: tests/run_all_ftp_tests.sh (hunk 2 above)
Comment (edwardhartnett) [reply tests/run_all_ftp_tests.sh:23]: Not easily.

**Review by AlexanderRichert-NOAA — APPROVED** [review]

**Pull request merged**